src/clj/rems/db/applications.clj

```diff
@@ -944,7 +940,9 @@
   (not (nil? (users/get-user-attributes userid))))
 
 (defn- valid-form-inputs? [application-id]
-  (= :valid (form-validation/validate (get-form-for application-id))))
+  ;; TODO a bit ugly:
+  (let [user-id (:applicantuserid (first (db/get-applications {:id application-id})))]
```

Comment:
Maybe there could be a better (named) helper function for this.

Reply:
suggestions? `(applicant-of-application application-id)`?

Reply:
Or `get-applicant-of-application`?